src/main/java/hu/oe/nik/szfmv/automatedcar/model/WorldObject.java

```diff
@@ -151,6 +158,24 @@ public BufferedImage getImage() {
         return this.image;
     }
 
+    @Override
+    public int getReferenceX()
+    {
+        return this.referencePosition.getX();
+    }
+
+    @Override
+    public int getReferenceY()
+    {
+        return this.referencePosition.getY();
+    }
+
+    public void setReferencePosition(Position referencePosition)
+    {
+        this.referencePosition = referencePosition;
+    }
+    
+    @Deprecated
```

Comment:
Ez is kiyeetelhető, ha nincs használva

Reply:
Azért deprecated, mert lehet más csapatnál még bent van. Nem szeretnék olyat, hogy valami miatt ne forduljon a kód, majd a kövi sprintben/refaktnál töröljük.

Reply:
Szerintem a többi csapat nem hoz létre ilyet, és ha a mainben van ilyen akkor pedig előjön merge conflictként, de értem az aggodalmat, szerintem még sprint1-ben vegyük ki